source/features/tag-changelog-link.tsx

```diff
@@ -124,5 +120,8 @@ features.add({
 	include: [
 		pageDetect.isReleasesOrTags
 	],
+	exclude: [
+		() => select.exists('.blankslate') // The repo has no tags
```

Comment:
I meant this, sorry

Reply:
Not sure. Its not an empty repo, it anywhere where its a blank state. 
Ex. No issues, No pull requests.

Reply:
Right, it becomes “is empty repo” if you prepend it with `isRepo() &&`

Reply:
What about `isBlankRepoState`?